src/main/resources/com/google/api/codegen/py/main.snip

```diff
@@ -174,24 +170,27 @@
                 default_client_config,
                 client_config,
                 config.STATUS_CODE_NAMES,
-                kwargs={'metadata': metadata})
+                metrics_headers=metrics_headers,
+            )
         @end
     @end
 @end
 
 @private initMethodSection(service)
     @let ifaceConfig = context.getApiConfig.getInterfaceConfig(service)
-        def __init__(
-                self,
+        def __init__(self,
                 service_path=SERVICE_ADDRESS,
                 port=DEFAULT_SERVICE_PORT,
                 channel=None,
                 credentials=None,
                 ssl_credentials=None,
                 scopes=None,
                 client_config=None,
-                app_name='gax',
-                app_version=_GAX_VERSION):
+                app_name=None,
+                app_version='0.0',
+                lib_name=None,
+                lib_version='0.0',
```

Comment:
The `gax` portion of the header is added [here][1].

  [1]: https://github.com/lukesneeringer/gax-python/blob/headers/google/gax/api_callable.py#L341